x/sync/client_test.go

```diff
@@ -263,7 +262,6 @@ func TestGetRangeProof(t *testing.T) {
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			require := require.New(t)
```

Comment:
this is needed since the inner test will panic if it fails on a call that captures the outer `t`, I believe